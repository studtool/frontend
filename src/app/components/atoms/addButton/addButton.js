import React from 'react';

export function AddButton (props){
    const {
        qa = false,
        title = '',
        handleClick = null,
    } = props;
    return (
        <button onClick={handleClick}>
            +
        </button>
    );
}
