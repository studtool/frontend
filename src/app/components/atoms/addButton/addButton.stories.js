import * as React from 'react';
import {storiesOf} from '@storybook/react';
import {AddButton} from 'App/components/atoms/addButton/addButton';

storiesOf('AddButton', module)
    .add('AddButton default', () => (
        <AddButton/>
    ));
