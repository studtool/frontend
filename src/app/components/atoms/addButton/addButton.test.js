import React from 'react';
import {AddButton} from './addButton';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

test('button test', () => {
    const tree = renderer.create(<AddButton/>).toJSON();

    expect(tree).toMatchSnapshot();
});
