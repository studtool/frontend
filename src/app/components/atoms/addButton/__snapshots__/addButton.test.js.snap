// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button test 1`] = `
<button
  onClick={null}
>
  +
</button>
`;
